database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java

```diff
@@ -123,16 +138,38 @@ private Connection getConnection(Configuration conf) {
     Connection connection;
     try {
       String url = conf.get(DBConfiguration.URL_PROPERTY);
+      if (url.contains("com.google.cloud.sql.mysql.SocketFactory")) {
+        delegateClass = "com.google.cloud.sql.mysql.SocketFactory";
+        socketFactory = "io.cdap.socketfactory.mysql.SocketFactory";
+        url = url.replace("com.google.cloud.sql.mysql.SocketFactory",
```

Comment:
Can use delegate class instead of string or they can all be turned into private statics no ?

Reply:
Changed to private static.